modules/core/src/main/java/org/apache/ignite/internal/managers/encryption/GridEncryptionManager.java

```diff
@@ -311,6 +311,8 @@ public GridEncryptionManager(GridKernalContext ctx) {
             }
         });
 
+        ctx.internalSubscriptionProcessor().registerGlobalStateListener(this);
```

Comment:
`GridTaskProcessor` is a `GridComponent`. Its `onKernalStart` is already called by `IgniteKernal#start()`, `for (GridComponent comp : ctx){ comp.onKernalStart(active). }`. Do we need to keep one of them? Do we need `implements IgniteChangeGlobalStateSupport` for `GridTaskProcessor`?